packages/react-devtools-shared/src/devtools/store.js

```diff
@@ -416,6 +416,11 @@ export default class Store extends EventEmitter<{|
   }
 
   getElementByID(id: number): Element | null {
+    if (
+      this._profilerStore._dataFrontend &&
+      this._profilerStore._dataFrontend.imported === true
```

Comment:
This is not right.

There are several places in the Components tab that call this method, not just the Profiler. This would break (disable) all of the uses in Components anytime you happened to have profiling data imported in  the Profiler tab.

Reply:
I thought that was the main point, to disable selection on the Components tab?

Reply:
No. The Components tab should still work for your current app. So things should still be able to ask for info about a particular component.

The Profiler tab just shouldn't expect to load a component from the `Store` for a previous session. So this check needs to be in the Profiler, where it calls `store.getElementByID`